.gitignore

```
@@ -1,2 +1,5 @@
 _output
 release.yaml
+kubeconfig
+smoke/features/steps/__pycache__
```

Would it make sense to omit all `__pycache__` directories?
```suggestion
__pycache__/
```

---

Not sure about that, but i can only see __pycache__ in smoke/features/steps